test/modules/test_nuclei_dast.py

```diff
@@ -0,0 +1,47 @@
+from test.base import ArtemisModuleTestCase
+
+from karton.core import Task
+
+from artemis.binds import Service, TaskStatus, TaskType
+from artemis.modules.nuclei_dast import NucleiDast
+
+
+class NucleiDastTest(ArtemisModuleTestCase):
+    # The reason for ignoring mypy error is https://github.com/CERT-Polska/karton/issues/201
+    karton_class = NucleiDast  # type: ignore
+
+    def test_lfi_dast_template(self) -> None:
+        task = Task(
+            {"type": TaskType.SERVICE.value, "service": Service.HTTP.value},
+            payload={
+                "host": "test-flask-vulnerable-api",
+                "path": "/files",
+                "params": {"filename": "randomfilename"},
```

Comment:
that will fail in real production usage, as the 'params' variable will never be filled.

I think the module itself should be responsible for filling the variable (the code for generating params is present e.g. in the xss_scanner module, it can be extracted into a function)